src/OrleansSQLUtils/CreateOrleansTables_MySql.sql

```diff
@@ -58,8 +58,8 @@ CREATE TABLE OrleansMembershipTable
 	ProxyPort INT NULL,
 	SuspectTimes VARCHAR(8000) NULL,
 	StartTime DATETIME NOT NULL,
-	IAmAliveTime DATETIME NOT NULL,	
-	
+	IAmAliveTime DATETIME NOT NULL,
+
```

Comment:
I know it's extra work, but I think that only the real changes should appear in the diff. All of the editor added noise should be removed to ease reviewing of such a huge PR.